src/main/java/net/earthcomputer/clientcommands/command/KitCommand.java

```diff
@@ -0,0 +1,182 @@
+                        .then(argument("name", StringArgumentType.string())
+                                .executes(ctx -> create(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("delete")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .executes(ctx -> delete(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("edit")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .executes(ctx -> edit(ctx.getSource(), StringArgumentType.getString(ctx, "name")))))
+                .then(literal("kit")
+                        .then(argument("name", StringArgumentType.string())
+                                .suggests((ctx, builder) -> CommandSource.suggestMatching(kits.keySet(), builder))
+                                .then(literal("--override")
+                                        .executes(ctx -> kit(ctx.getSource(), StringArgumentType.getString(ctx, "name"), true)))
+                                .executes(ctx -> kit(ctx.getSource(), StringArgumentType.getString(ctx, "name"), false))))
+                .then(literal("kits")
+                        .executes(ctx -> kits(ctx.getSource()))));
+    }
+
+    private static int create(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.put(name, client.player.inventory) == null) {
+            saveFile();
+            sendFeedback("commands.ckit.create.success", name);
+        } else {
+            saveFile();
+            sendFeedback("commands.ckit.create.alreadyExists", name);
+        }
+        return 1;
+    }
+
+    private static int delete(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.remove(name) == null) throw NOT_FOUND_EXCEPTION.create(name);
+        saveFile();
+        sendFeedback("commands.ckit.delete.success", name);
+        return 1;
+    }
+
+    private static int edit(ServerCommandSource source, String name) throws CommandSyntaxException {
+        loadFile();
+
+        if (kits.put(name, client.player.inventory) == null) {
+            saveFile();
+            sendFeedback("commands.ckit.edit.notFoundSoCreatedNew", name);
+        } else {
+            saveFile();
+            sendFeedback("commands.ckit.edit.success", name);
+        }
+        return 1;
+    }
+
+    private static int kit(ServerCommandSource source, String name, boolean override) throws CommandSyntaxException {
+        if (!client.player.isCreative() || !client.player.abilities.creativeMode) throw NOT_CREATIVE_EXCEPTION.create();
```

Comment:
New code in clientcommands should use braces for all if statements. There are a few of them in this PR.